### ISSUE TITLE
Static file data sources

### ISSUE THREAD

**Issue by leoyvens** [graphprotocol/graph-node#3864]
The blocker to supporting static file data sources is that static data sources are not stored in the DB, and therefore we would not be able to update their status when the file is found.

**Comment by azf20** [comment]
@leoyvens static in this case meaning configured as part of `subgraph.yaml`?

**Comment by leoyvens** [comment]
@azf20 yes, in the manifest under the `dataSources` section with a specified `source`.

**Comment by leoyvens** [comment]
Still relevant.
